reflekt-dsl/src/main/kotlin/org/jetbrains/reflekt/ReflektClass.kt

```diff
@@ -0,0 +1,44 @@
+package org.jetbrains.reflekt
+
+import kotlin.reflect.*
+
+/**
+ * Represents a class and designed for storing information about it, which is collected by Reflekt at compile-time.
+ *
+ * @param T the type of the class.
+ * @property kotlin [KClass] instance for this class.
```

Comment:
`kClass`? `kotlin` looks a little bit strange for me..

Reply:
OK